operator/pkg/scheme/scheme.go

```diff
@@ -0,0 +1,17 @@
+package scheme
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	operatorv1alpha1 "github.com/karmada-io/karmada/operator/pkg/apis/operator/v1alpha1"
+)
+
+// Scheme holds the aggregated Kubernetes's schemes and extended schemes.
+var Scheme = runtime.NewScheme()
```

Comment:
@lonelyCZ should I reuse the `gclient`package ?

https://github.com/karmada-io/karmada/blob/master/pkg/util/gclient/gclient.go#L40

Reply:
I think we only need to use the specific resource, what resources will it operate?

Reply:
The operator only requires the karmada and some kubernetes resource. I'm not sure if we reuse the existing scheme in gclient package.

Reply:
Avoiding confusion between karmada built-in resources and operators, I think we can redefine a client that only has necessary scheme.

and cc @RainbowMango

Reply:
makes sense to me.